addon/components/document-details.hbs

```diff
@@ -77,15 +77,20 @@
         </span>
       </span>
 
-      <p class="uk-flex">
+      <p class="uk-margin-remove-top">
+        <div class="uk-text-meta">
```

Comment:
Isn't that more like a `<label>` for the textarea?

Reply:
Nice idea! :sunglasses: I like!

Reply:
Done